docs/app/pages/components/components-sections/input-controls/typeahead/snippets/app.html

```diff
@@ -0,0 +1,27 @@
+<div class="row">
+    <div class="col-md-12 has-feedback">
+        <form class="input-typeahead-form">
+            <input [value]="value"
+                   [(ngModel)]="input"
+                   [ngModelOptions]="{standalone: true}"
+                   [placeholder]="'Enter Text'"
+                   class="form-control"
+                   (keydown.enter)="dropdownOpen = true; $event.preventDefault()"
+                   (click)="dropdownOpen = true; $event.stopPropagation();"
+                   (keydown.escape)="dropdownOpen = false; $event.stopPropagation();"
+                   [attr.aria-controls]="typeahead">
+            <ux-typeahead #typeahead
+                          class="typeahead-example"
+                          [open]="dropdownOpen"
+                          [filter]="input"
+                          [options]="loadOptionsFn"
+                          [openOnFilterChange]="true"
+                          [pageSize]="5"
```

Comment:
We should make the default page size larger, currently there is a flicker when the typeahead opens and the scrollbar jumps several times as it has to load multiple pages to fill the height of the typeahead.